### ISSUE TITLE
Toggle update

### ISSUE THREAD

**Issue by cathystoli** [chanzuckerberg/sci-components#692]
**Issue:**
1. SR reads 'checkbox not checked' which is incorrect control type being announced

2. SR does not read what is being toggled on or toggled off (no additional context)

3. Cannot toggle component on or off via keyboard

4. No focus control indicator

**Solution:**
1 & 2. SR should read something like 'toggle on' or 'toggle off' along with the label

3. Allow component to be utilized via keyboard controls

4. Add focus control indicator to component
